proto/common/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+// This module contains constants related to remote protobuf files as well as helpers for referencing them.
+// This also helps manage interface versions from a central location, which is particularly helpful for Ibeji since
+// the interfaces are referenced in two different crates.
+
+pub const GITHUB_BASE_URL: &str = "https://raw.githubusercontent.com";
+
+pub mod freyja {
+    pub const REPO_NAME: &str = "eclipse-ibeji/freyja";
+    pub const VERSION: &str = "0.1.0";
```

Comment:
REPO_VERSION

Reply:
this is the same convention as the Freyja repo, and the `interface_url!` macro won't work if we rename this

Reply:
Okay.  Leave as is then.